src/main/k/working/search/tests/initial.star.initial.output.golden

```diff
@@ -0,0 +1,3 @@
+VarDotVar0 ==K <generatedCounter>
```

Comment:
I was confused by this at first, but I think this is due to the fact that your `pattern` gets completed to a full configuration to allow matching on it.  I wonder whether it would be a good idea to eliminate `DotVar`s from the substitution before we convert it back to a pattern.   Not necessarily in this PR, though :-)